pkg/webhook/validation/uservalidation.go

```diff
@@ -208,3 +222,15 @@ func ValidateMCIdentity(ctx context.Context, client client.Client, req admission
 	klog.V(2).InfoS(deniedModifyResource, "user", userInfo.Username, "groups", userInfo.Groups, "operation", req.Operation, "GVK", req.RequestKind, "subResource", req.SubResource, "namespacedName", namespacedName)
 	return admission.Denied(fmt.Sprintf(resourceDeniedFormat, userInfo.Username, userInfo.Groups, req.Operation, req.RequestKind, req.SubResource, namespacedName))
 }
+
+func isFleetV1Beta1API(ctx context.Context, client client.Client) (bool, error) {
+	var crd apiextensionsv1.CustomResourceDefinition
+	v1Beta1MCCRDName := "memberclusters.cluster.kubernetes-fleet.io"
```

Comment:
Hi Arvind! This should be a constant.

Reply:
Another optimization we can have is that, since the API version is immutable after hub cluster creation, we do not need to repeatedly call this method.